pkg/api/testing/fuzzer.go

```diff
@@ -382,11 +382,16 @@ func FuzzerFor(t *testing.T, version schema.GroupVersion, src rand.Source) *fuzz
 			}
 			if c.RandBool() {
 				c.Fuzz(&ev.ConfigMapRef)
+			} else if c.RandBool() {
```

Comment:
you can only have ConfigMapRef or SecretRef.
At best it could be rewritten as
```
if c.RandBool() {
} else {
  if c.RandBool() {
  }
}
```

Although the more I think about it, this should be just a plain else. I will fix it... I got lucky and didn't fail fuzzing with no entries.